templates/default/netuitive-agent.conf.erb

```diff
@@ -148,8 +148,7 @@ include_cpu_pct = True
 
 [[DiskSpaceCollector]]
 enabled = True
-# exclude everything that begins /boot or /mnt
-exclude_filters = ^/boot, ^/mnt
+exclude_filters = <%= @disk_space_collector_exclude_filters %>
```

Comment:
so I don't really know enough about netuitive anymore, it looks like according to the default it uses a comma separated list of regexes. My question is if it makes sense to manage it as an array and then join them here rather than managing it as a string? We can optionally support both if its easier to manage for some as a string.

Reply:
@majormoses is there an advantage to managing this as an array rather than as a string?

Reply:
I think it depends on a couple of things:
- how sensitive is the parser to spaces, double spaces, or lack of a space surrounding commas? If so then this helps prevent a simple typo
- the number of regexes that someone would want/need. If they number could likely be more than 5 I think we should at least enable it as an option as it is less likely to typos being word wrapped across multiple lines in your editor.

Reply:
@majormoses that makes sense. I don't _think_ the parser is sensitive to spaces surrounding commands, but @rclagett might know. I don't think I've seen more than one or two scenarios where people are adjusting the exclude_filters for this collector, but that doesn't mean there couldn't be a situation where someone needed a whole bunch. I'd say the best option would be to support both with string as the default.

Reply:
Awesome! Thanks @majormoses !